ml.prevete.Daty.appdata.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>ml.prevete.Daty</id>
+  <launchable type="desktop-id">ml.prevete.Daty.desktop</launchable>
+  <name>Daty</name>
+  <summary>Free Wikidata editor.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <description>
+    <p>
+      <b>Daty is a Wikidata editor</b>
```

Comment:
Something a bit better here would be good. How about this (lightly adapted from the README)
> Daty is a free Wikidata editor adhering to GNOME Human Interface Guidelines, intended to enable a better editing workflow and faster deployment of requested user features for Wikidata.
Use Daty to search, select, read, batch edit items, script actions, share, visualize proposed changes and bots.

Reply:
Thank you!